src/experiments/logins/api.js

```diff
@@ -63,6 +65,19 @@ this.logins = class extends ExtensionAPI {
               throw new ExtensionError(ex);
             }
           },
+          setManagementURI(url) {
+            try {
+              const prefs = Services.prefs.getDefaultBranch(null);
+              prefs.setStringPref(PREF_MANAGEMENT_OVERRIDE, url);
```

Comment:
I'd probably just do `Services.prefs.setStringPref()` directly, and not bother getting the branch on the line above.

Reply:
The `getDefaultBranch()` call was recommended by MattN to help minimize the amount of cleanup the extension would need to do.

Reply:
TIL :-)